src/rest-server/src/config/job.js

```diff
@@ -105,6 +105,11 @@ const jobConfigSchema = Joi.object().keys({
         /\w+/,
         Joi.required()
     ),
+  extras: Joi.object()
+    .pattern(
+        /\w+/,
```

Comment:
`/\w+/` means every keys should be made up of `[A-Za-z0-9_]`, it's better to add dot(.) at least for namespaced keys.